includes/class-convertkit-api.php

```diff
@@ -114,7 +114,7 @@ public function update_resources( $api_key, $api_secret ) {
 
 		// Forms and Landing Pages.
 		$api_response = $this->_get_api_response( 'forms' );
-		WP_ConvertKit::log( 'API Response: ' . print_r( $api_response, true ) );
+		WP_ConvertKit::log( 'API Response: Success (Returned ' . count($api_response['forms']) . " forms.)");
```

Comment:
Will this work as expected if `$api_response['forms']` is null, or if `$api_response` is not an object?